ParkingTicketFrontEnd/Extensions/ParkingOffenseFactory.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using ParkingTicketLogic;
+
+namespace ParkingTicketFrontEnd.Extensions
+{
+    public class ParkingOffenseFactory
+    {
+        /// <summary>
+        /// welcome to the DRY example of not repeating yourself
+        /// write the code once so that you have one place to go
+        /// when something goes wrong. ejw 2-JLY-2019
+        /// I ripped off this code from 
+        /// https://www.codeproject.com/Articles/1056011/Bind-Enum-to-DropdownList-in-ASP-NET-MVC
+        /// To create a quick and dirty means to pull enumerations into the UI 
+        /// </summary>
+        /// <returns>MVC Select List</returns>
+        public static SelectList CreateOffenseSelectList()
+        {
+            SelectList _outbound;
+            
+            //a bit of a hack but this will work
+            var offfenses = from ParkingOffense e in Enum.GetValues(typeof(ParkingOffense))
```

Comment:
I want you to add a "//Todo: " on there. We're going to extend this app further down the line. so maybe we want to make this method generic, so that no mater what type of enum it is, it spits out a SelectList.

Reply:
It will give me a chance to work with Generics and will make the code more extensible.  **I'LL DO IT!**

Reply:
I have created a generic method to handle this and it works pretty well.  How would I test this?  Also need it to extend to a selected item in a list.